np-gravity.py

```diff
@@ -117,6 +118,8 @@ def draw(screen, game):
                     game.update(UP)
                 elif event.key == pygame.K_DOWN:
                     game.update(DOWN)
+                elif event.key == pygame.K_SPACE:
```

Comment:
Is this to change grip in Megalit only?

Reply:
correct. i believe it won't cause any issues in popils